app/models/ext_management_system.rb

```diff
@@ -92,6 +92,12 @@ def hostname_uniqueness_valid?
     errors.add(:hostname, N_("has to be unique per provider type")) if existing_hostnames.include?(hostname.downcase)
   end
 
+  def hostname_format_valid?
+    return if hostname.ipaddress? || hostname.hostname?
```

Comment:
It would be a good idea to update the Gemfile setting `more_core_extensions` to `'~> 3.5'`https://github.com/ManageIQ/manageiq/blob/master/Gemfile#L49

Reply:
It's done :+1: